examples/COUPLE/lammps_mc/mc.cpp

```diff
@@ -0,0 +1,261 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+------------------------------------------------------------------------- */
+
+// MC code used with LAMMPS in client/server mode
+// MC is the client, LAMMPS is the server
+
+// Syntax: mc infile mode modearg
+//         mode = file, zmq
+//         modearg = filename for file, localhost:5555 for zmq
+
+#include <math.h>
```

Comment:
@sjplimp for consistency, these include statements should use c++ style headers like `<cmath>` and alike